lib/broccoli/ember-app.js

```diff
@@ -658,6 +665,23 @@ EmberApp.prototype.index = function() {
 };
 
 /**
+  Filters a given app tree ('app' or 'fastboot/app') with excluding templates and any other patterns specified.
+
+  @private
+  @method _filterAnyAppTree
+  @return Tree
+*/
+EmberApp.prototype._filterAnyAppTree = function (treeToFilter, appExcludePatterns, annotationTree) {
```

Comment:
I'd generally rather have the people who need to filter use the full power of Broccoli as opposed to this helper on top of `Funnel`.